tests/test_e2e_70_kytos_stats.py

```diff
@@ -0,0 +1,237 @@
+
+        api_url = KYTOS_STATS + '/flow/stats'  
+        response = requests.get(api_url)
+        assert response.status_code == 200, response.text
+        data = response.json()
+
+        api_url = KYTOS_API + '/kytos/topology/v3/switches'
+        response = requests.get(api_url)
+        data_topp = response.json()
+        assert len(data) == len(data_topp['switches'])
+
+    def test_010_table_stats(self):
+        """Test table_stats""" 
+
+        api_url = KYTOS_STATS + '/table/stats?dpid=00:00:00:00:00:00:00:01' 
+        response = requests.get(api_url)
+        assert response.status_code == 200, response.text
+        data = response.json()
+        assert len(data) == 1
+        assert '00:00:00:00:00:00:00:01' in data
+
+        api_url = KYTOS_STATS + '/table/stats?dpid=00:00:00:00:00:00:00:01&dpid=00:00:00:00:00:00:00:02' 
+        response = requests.get(api_url)
+        assert response.status_code == 200, response.text
+        data = response.json()
+        assert len(data) == 2
+        assert '00:00:00:00:00:00:00:01' in data
+        assert '00:00:00:00:00:00:00:02' in data
+
+        api_url = KYTOS_API + '/kytos/topology/v3/switches'
+        response = requests.get(api_url)
+        data_topp = response.json()
+        topo_switches = data_topp['switches']
+
+        api_url = KYTOS_STATS + '/table/stats?table=0'  
+        response = requests.get(api_url)
+        assert response.status_code == 200, response.text
+        data = response.json()
+        assert len(data) == len(topo_switches)
+        for sw in topo_switches:
+            assert len(data[sw]) == 1
+            assert '0' in data[sw]
+
+        api_url = KYTOS_STATS + '/table/stats?table=0&table=1'  
+        response = requests.get(api_url)
+        assert response.status_code == 200, response.text
+        data = response.json()
+        assert len(data) == len(topo_switches)
+        for sw in topo_switches:
+            assert len(data[sw]) == 2
+            assert '0' in data[sw]
+            assert '1' in data[sw]
+
+        api_url = KYTOS_STATS + '/table/stats?dpid=00:00:00:00:00:00:00:01&table=0'
+        response = requests.get(api_url)
+        assert response.status_code == 200, response.text
+        data = response.json()
+        assert len(data) == 1
+        assert len(data['00:00:00:00:00:00:00:01']) == 1
+        assert '0' in data['00:00:00:00:00:00:00:01']
+
+        api_url = KYTOS_STATS + '/table/stats'  
+        response = requests.get(api_url)
+        assert response.status_code == 200, response.text
+        data = response.json()
+        assert len(data) == len(topo_switches)
+
+    def test_015_packet_count(self):
+        """Test packet_count""" 
+
+        api_url = KYTOS_STATS + '/flow/stats'  
+        response = requests.get(api_url)
+        assert response.status_code == 200, response.text
+        data_flow = response.json()
+        for sw in data_flow:
+            if len(data_flow[sw]):
```

Comment:
Instead of trying to find a non-empty flow, maybe we could add such flow and query it later for stats.

Reply:
@italovalcy, it seems very good to me, what do you think about adding that test as part of the new test and leaving this test as it is?

Reply:
@italovalcy, I created an issue to add all your suggestions and close this PR here.